ui/test/nuxeo-error.test.js

```diff
@@ -0,0 +1,119 @@
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import { fixture, flush, html, isElementVisible } from '@nuxeo/nuxeo-elements/test/test-helpers.js';
+import '../nuxeo-error.js';
+/* eslint-disable no-unused-expressions */
+
+suite('<nuxeo-error>', () => {
+  const testNuxeoErrorVisibility = (nuxeoError) => {
+    expect(isElementVisible(nuxeoError)).to.be.true;
+
+    const code = nuxeoError.shadowRoot.querySelector('.code');
+    expect(isElementVisible(code)).to.be.true;
+    expect(code.innerText).to.equal('404');
+
+    const description = nuxeoError.shadowRoot.querySelector('.description');
+    expect(isElementVisible(description)).to.be.true;
+    expect(description.innerText).to.equal('ERROR.404');
+
+    const url = nuxeoError.shadowRoot.querySelector('.url');
+    expect(isElementVisible(url)).to.be.true;
+    expect(url.innerText).to.equal('notfound.html');
+
+    const message = nuxeoError.shadowRoot.querySelector('.message');
+    expect(isElementVisible(message)).to.be.true;
+    expect(message.innerText).to.equal('Failed to find layout');
+  };
+
+  test('Should display error messages when properties are set', async () => {
+    const nuxeoError = await fixture(
+      html`
+        <nuxeo-error code="404" message="Failed to find layout" url="notfound.html"></nuxeo-error>
+      `,
+    );
+    await flush();
+
+    testNuxeoErrorVisibility(nuxeoError);
+  });
+
+  test('Should not display error messages when properties are empty', async () => {
+    const nuxeoError = await fixture(
+      html`
+        <nuxeo-error code="" message="" url=""></nuxeo-error>
+      `,
+    );
+    await flush();
+
+    expect(isElementVisible(nuxeoError)).to.be.true;
+
+    const code = nuxeoError.shadowRoot.querySelector('.code');
+    expect(isElementVisible(code)).to.be.false;
+    expect(code.innerText).to.be.empty;
+
+    const description = nuxeoError.shadowRoot.querySelector('.description');
+    expect(isElementVisible(description)).to.be.false;
+    expect(description.innerText).to.be.empty;
+
+    const url = nuxeoError.shadowRoot.querySelector('.url');
+    expect(isElementVisible(url)).to.be.false;
+    expect(url.innerText).to.be.empty;
+
+    const message = nuxeoError.shadowRoot.querySelector('.message');
+    expect(isElementVisible(message)).to.be.false;
+    expect(message.innerText).to.be.empty;
+  });
+
+  test('Should display error messages when "show" function is run', async () => {
+    const nuxeoError = await fixture(
+      html`
+        <nuxeo-error code="404" message="Failed to find layout" url="notfound.html"></nuxeo-error>
+      `,
+    );
+    await flush();
+
+    nuxeoError.show();
+    testNuxeoErrorVisibility(nuxeoError);
+  });
+
+  test('Should display error messages when "show" function is run with arguments', async () => {
+    const nuxeoError = await fixture(
+      html`
+        <nuxeo-error></nuxeo-error>
```

Comment:
I believe this should have the same fixture of `Should not display error messages when properties are empty`, otherwise, we should be checking before calling `show` that the properties were not already visible.